pyproject.toml

```
@@ -36,7 +36,7 @@ sunspecdemo = { git = "https://github.com/epcpower/sunspec-demo.git", rev = "a04
 ###################
 
 attrs = "20.2.0"
-lxml = "4.3.0"
+lxml = "4.5.0"
```

Revert back. Not for this PR.
```suggestion
lxml = "4.3.0"
```

---

Done.